### PR TITLE
Replaced url  fairlearn.github.io  fairlearn.org

### LINKED ISSUE #742: Update links from fairlearn.github.io to fairlearn.org
#### Describe the issue linked to the documentation

A variety of our documentation links point to `fairlearn.github.io` rather than `fairlearn.org`. There is a redirect, but it would be good to fix these

#### Suggest a potential alternative/fix

See above

### PR DESCRIPTION
**Pull request by BrandonGoding**
replaced fairlearn.github.io.git with fairlearn.git in files with standard url.  in config.yml replaced fairlearn.github.io.git with fairlearn.git

Fixes #742

### CODE REVIEW

**Comment by adrinjalali** [comment]
Hi @BrandonGoding , I updated the PR description to link it to the issue.

To fix the DCO issue, you can click on the "Detail" button, and follow the instructions there :)

File: .circleci/config.yml [hunk 1]
```diff
@@ -107,7 +107,7 @@ jobs:
 
       - run:
           name: 'Push to GitHub Pages repository'
-          command: gh-pages --dotfiles --message "Updates" --dist docs/_build/html --repo https://github.com/fairlearn/fairlearn.github.io.git --branch master
+          command: gh-pages --dotfiles --message "Updates" --dist docs/_build/html --repo https://github.com/fairlearn/fairlearn.git --branch master
```
Comment (riedgar-ms) [inline .circleci/config.yml:110]: I expect this one needs to be left in.

File: docs/static_landing_page/index.html [hunk 2]
```diff
@@ -9,7 +9,7 @@
 		<meta property="og:type" content="website"/>
 		<meta property="og:description" content="An open source toolkit for assessing and improving fairness in AI">
 		<meta property="og:image" content="./images/fairlearn-meta-image.jpg">
-		<meta property="og:url" content="http://fairlearn.github.io">
+		<meta property="og:url" content="http://fairlearn.org">
```
Comment (riedgar-ms) [inline docs/static_landing_page/index.html:12]: I'd not noticed this before, but this should also probably be `https`

**Review by riedgar-ms — COMMENTED** [review]
Thanks for picking this up so quickly!

File: CHANGES.md [hunk 3]
```diff
@@ -54,7 +54,7 @@
     descriptive name and for consistency with the paper. Similarly,
     `AverageLossMoment` renamed to `MeanLoss`.
   For a comprehensive overview of available constraints refer to the new [user
-  guide on fairness constraints for reductions methods](https://fairlearn.github.io/user_guide/mitigation.html#reductions).
+  guide on fairness constraints for reductions methods](https://fairlearn.org/user_guide/mitigation.html#reductions).
```
Comment (riedgar-ms) [inline CHANGES.md:57]: This should be:
https://fairlearn.org/main/user_guide/mitigation.html#reductions

File: docs/contributor_guide/developer_call_notes/2020_06_11.rst [hunk 4]
```diff
@@ -21,7 +21,7 @@ Attendees:
 
 Agenda:
 
-- new website at https://fairlearn.github.io/
+- new website at https://fairlearn.org/
```
Comment (riedgar-ms) [inline docs/contributor_guide/developer_call_notes/2020_06_11.rst:24]: Probably shouldn't change these bits of history in the call notes

**Pull request closed without merging**